test/Generators/Microsoft.Gen.Logging/Unit/ParserTests.cs

```diff
@@ -195,7 +195,7 @@ public async Task UnderscoresInMethodName()
                 partial class C
                 {
                     [LoggerMessage(0, LogLevel.Debug, ""M1"")]
-                    static partial void __M1(ILogger logger);
+                    static partial void /*0+*/__M1/*-0*/(ILogger logger);
```

Comment:
What's this change about?

Reply:
It's unneccessary now. The base generator does not allow underscores, but we can.